CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contribute
+
+## Install
+
+```console
+yarn install
+```
+
+## Test
+
+**requirements:** [`just`][just]
+
+Pass a `feature` and a `base image`, _e.g._:
+
+```fish
+just test fish mcr.microsoft.com/devcontainers/base:debian
```

Comment:
There is also https://github.com/go-task/task which looks good. Before committing to something, we should compare those (sorry for being pedantic, I'd like those decisions to be organization-wide so it requires a certain level of scrupulousness).